packages/shared/src/UI/components/WalletStatusBar/WalletBarButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { useCurrentWeb3NetworkPluginID } from '@masknet/plugin-infra/web3'
+import { makeStyles, MaskColorVar, useStylesExtends } from '@masknet/theme'
+import { Button, CircularProgress } from '@mui/material'
+import classNames from 'classnames'
+import { useState } from 'react'
+import { useSharedI18N } from '../../../locales'
+import type { WalletButtonActionProps } from './types'
+
+interface WalletButtonProps extends withClasses<'root'> {
+    actionProps?: WalletButtonActionProps
+    onConnectWallet?: () => void
+}
+
+const useStyles = makeStyles<{ color?: 'warning' }>()((theme, props) => ({
+    progress: {
+        color: MaskColorVar.twitterButtonText,
```

Comment:
We don't use `MaskColorVar` anymore. Use `theme.palette.maskColor` instead. You can consult @nuanyang233 for more information.